Frameworks/CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -498,6 +498,7 @@ CF_INLINE CFRange CFRangeMake(CFIndex loc, CFIndex len) {
 CF_EXPORT
 CFRange __CFRangeMake(CFIndex loc, CFIndex len);
 
+#define NSMakeRangeFromCF(cfr) NSMakeRange(cfr.location == kCFNotFound ? NSNotFound : cfr.location, cfr.length)
```

Comment:
Going to keep it local to CT for now until I find a good place for it

---
In reply to: [77923909](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77923909) [](ancestors = 77923909,77900982)